### PR TITLE
Describe content of the staging workflow table

### PR DESCRIPTION
**Pull request by krauselukas**
screenshot-2018-11-22 using the obs web ui best practice guide
I described the table content a little bit

### CODE REVIEW

File: xml/obs_best_practice_staging_workflow.xml [hunk 1]
```diff
@@ -49,34 +49,70 @@
...
-          List of requests that were in the backlog and have a review accepted.
-        </para>
+      <para>
+        Table content:
+      </para>
+        <itemizedlist>
+          <listitem>
+            <para>
+              <emphasis>Staging Project:</emphasis>
+              Shows the staging project name, it's overall state (see legend),
```
Comment (dmarcoux) [inline xml/obs_best_practice_staging_workflow.xml]: **its** overall state

File: xml/obs_best_practice_staging_workflow.xml [hunk 2]
```diff
@@ -49,34 +49,70 @@
...
+              and the overall build progress of the packages within the project.
+            </para>
+          </listitem>
+          <listitem>
+            <para>
+              <emphasis>Requests:</emphasis>
+              Shows the different requests associated and the current state of them.
```
Comment (dmarcoux) [inline xml/obs_best_practice_staging_workflow.xml]: Show the associated requests and their current state

File: xml/obs_best_practice_staging_workflow.xml [hunk 3]
```diff
@@ -49,34 +49,70 @@
...
+            </para>
+          </listitem>
+          <listitem>
+            <para>
+              <emphasis>Problems:</emphasis>
+              Shows build problems of packages within the project and status problems
+              from external services like openQA sended to the status api of the buildservice.
```
Comment (dmarcoux) [inline xml/obs_best_practice_staging_workflow.xml]: [...] and status problems reported to the Build Service's Status API by external services like openQA.

File: xml/obs_best_practice_staging_workflow.xml [hunk 4]
```diff
@@ -49,34 +49,70 @@
...
+          <listitem>
+            <para>
+              <emphasis>Managers:</emphasis>
+              Shows the group that are the Staging Mangers
+            </para>
+          </listitem>
+          <listitem>
+            <para>
+              <emphasis>Empty projects:</emphasis>
+              Staging projects without and requests assigned.
```
Comment (dmarcoux) [inline xml/obs_best_practice_staging_workflow.xml]: Staging projects without assigned requests.

File: xml/obs_best_practice_staging_workflow.xml [hunk 5]
```diff
@@ -49,34 +49,70 @@
...
+          <listitem>
+            <para>
+              <emphasis>Backlog:</emphasis>
+              List of requests that can be assigned to a staging project.
+            </para>
+          </listitem>
+          <listitem>
+            <para>
+              <emphasis>Ready:</emphasis>
+              List of requests that were in the backlog and have a review accepted.
```
Comment (dmarcoux) [inline xml/obs_best_practice_staging_workflow.xml]: [...] have an accepted review.

File: xml/obs_best_practice_staging_workflow.xml [hunk 6]
```diff
@@ -49,34 +49,70 @@
...
+            </para>
+          </listitem>
+          <listitem>
+            <para>
+              <emphasis>Ignored:</emphasis>
+              List of requests excluded from this Staging.
```
Comment (dmarcoux) [inline xml/obs_best_practice_staging_workflow.xml]: from this staging workflow.

**Comment by krauselukas** [comment]
@dmarcoux done!

File: xml/obs_best_practice_staging_workflow.xml [hunk 7]
```diff
@@ -49,34 +49,70 @@
...
         <para>
-          <emphasis>Ignored:</emphasis>
-          List of requests excluded from this Staging.
+          Info section:
         </para>
+        <itemizedlist>
+          <listitem>
+            <para>
+              <emphasis>Managers:</emphasis>
+              Shows the group that are the Staging Mangers
```
Comment (dmarcoux) [inline xml/obs_best_practice_staging_workflow.xml]: Shows the Staging Man**a**gers group

**Comment by dmarcoux** [comment]
It's all good except for [this comment](https://github.com/openSUSE/obs-docu/pull/116#discussion_r235777492)

File: xml/obs_best_practice_staging_workflow.xml (hunk 7 above)
Comment (krauselukas) [reply xml/obs_best_practice_staging_workflow.xml]: @dmarcoux next time i will look a bit closer ;)

**Comment by krauselukas** [comment]
@dmarcoux done

**Pull request merged**